### PR TITLE
Expose parse_string and parse_file at top-level

### PR DESCRIPTION
**Pull request by denizdogan**
I use parse_string way more often than parse_file, so I would suggest this change :)

### CODE REVIEW

**Comment by denizdogan** [comment]
Forgot to expose parse_file and parse_string, but you get the gist :)

**Review by hassaans — APPROVED** [review]

File: ass/__init__.py [hunk 1]
```diff
@@ -10,4 +10,6 @@
     "parse",
 ]
```
Comment (FichteFoll) [inline ass/__init__.py:11]: You also need to adjust `__all__`.

**Review by FichteFoll — CHANGES_REQUESTED** [review]

**Comment by FichteFoll** [comment]
I personally barely used `parse_string`, but I agree it's weird to only expose one of them. If people were to interact with `Document` more frequently, I'd say they are more likely to `from ass import Document` and then use `Document.parse_file`, but realistically most use cases will probably remain `import ass; ass.parse…`, so I think this is a fine addition.

**Comment by denizdogan** [comment]
I think I deleted the branch where I put the PR and I can't figure out how to add to it now. If someone with commit access to this repo could just make these changes, I'd appreciate that. Otherwise, I'm going to have to make a new PR.

**Comment by FichteFoll** [comment]
That's fine with me, I'll just leave this PR open as an issue.

**Pull request closed without merging**